sdcm/utils/replication_strategy_utils.py

```diff
@@ -188,12 +190,15 @@ def _alter_keyspace_rf(self, keyspace: str, replication_factor: int, session):
             LOGGER.error(f"{message} Failed with: {error}")
             raise error
 
-    def revert_to_original_keyspaces_rf(self):
-        LOGGER.debug(f"Reverting keyspaces replication factor to original value of {self.datacenter}..")
-        with self.cluster.cql_connection_patient(self.cluster.nodes[0]) as session:
-            for keyspace in self.decreased_rf_keyspaces:
-                self._alter_keyspace_rf(keyspace=keyspace, replication_factor=self.original_nodes_number,
-                                        session=session)
+    def revert_to_original_keyspaces_rf(self, wait_node_balance: 'BaseNode' = None):
```

Comment:
`wait_node_balance` sounds like a boolean, it's a bit surprising to see it's type is `BaseNode`

Reply:
fixed: `node_to_wait_for_balance`